src/cache.cpp

```diff
@@ -747,15 +747,16 @@ void Cache::do_vacuum()
 	run_sql("VACUUM;");
 }
 
-int Cache::cleanup_cache(std::vector<std::shared_ptr<RssFeed>> feeds,
+/// if requested, removes unreachable data stored in cache.
+/// returns a count of unreachable feeds and items.
```

Comment:
This looks good, but please move it to the header. I don't have a very strong case for why I started writing doc-comments in the header file, but let's keep it consistent at least. ;)

Reply:
How's that? :)

Reply:
All cool :+1: Thanks!